apps/wakunode2/app.nim

```diff
@@ -252,6 +291,17 @@ proc retrieveDynamicBootstrapNodes*(dnsDiscovery: bool, dnsDiscoveryUrl: string,
   debug "No method for retrieving dynamic bootstrap nodes specified."
   ok(newSeq[RemotePeerInfo]()) # Return an empty seq by default
 
+proc setupDyamicBootstrapNodes*(app: var App): AppResult[void] =
+  let dynamicBootstrapNodesRes = retrieveDynamicBootstrapNodes(app.conf.dnsDiscovery,
+                                                               app.conf.dnsDiscoveryUrl,
+                                                               app.conf.dnsDiscoveryNameServers)
+  if dynamicBootstrapNodesRes.isOk():
+    app.dynamicBootstrapNodes = dynamicBootstrapNodesRes.get()
+  else:
+    warn "2/7 Retrieving dynamic bootstrap nodes failed. Continuing without dynamic bootstrap nodes.", error=dynamicBootstrapNodesRes.error
```

Comment:
What is this 2/7?

Reply:
Step 2 of 7. See:

https://github.com/waku-org/nwaku/blob/73cbafa658295213298b06ef7b349683f0d1f858/apps/wakunode2/wakunode2.nim#L719-L721

Reply:
Ah, I see:D